CHANGELOG.md

```diff
@@ -15,7 +15,10 @@
   - 🚀 NEW: `utilities.each-key()` will pass each _key_ of a map
     through a given function
 - 🚀 NEW: Allow `utilities.add()` to merge data, instead of override
+- 🚀 NEW: Include source-maps with JS & CSS assets
```

Comment:
Is this user-facing? What does it mean in practice?

Reply:
These are the JS and CSS files that are served as part of Herman itself (not user-provided). I guess it's maybe more `INTERNAL` than `NEW` -- but it is a change in production for us to now include source-maps for the Herman assets themselves.

Reply:
Got it. Sounds good.